libyul/optimiser/SSATransform.cpp

```diff
@@ -85,19 +85,19 @@ void IntroduceSSA::operator()(Block& _block)
 
 				// Replace "let a := v" by "let a_1 := v  let a := a_1"
 				// Replace "let a, b := v" by "let a_1, b_1 := v  let a := a_1 let b := b_2"
-				auto loc = varDecl.location;
+				shared_ptr<DebugData const> debugData = varDecl.debugData;
```

Comment:
Another useless local. I'd remove it and use `debugDataOf(varDecl)` directly instead.

Same with `assignment.debugData` below.

Reply:
https://github.com/ethereum/solidity/pull/11294#discussion_r625609560